### PR TITLE
Fix @cached decorator export from fake @glimmer/tracking module

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Discovered during: https://github.com/embroider-build/embroider/issues/1132#issuecomment-1045280240

(I put `@cached` in the wrong spot :( )

### CODE REVIEW

**Review by lifeart — APPROVED** [review]

**Comment by lifeart** [comment]
@NullVoxPopuli looks like we could use https://github.com/embroider-build/embroider/blob/main/packages/compat/tests/route-split.test.ts as reference for test, in my understanding we could have minimal ember app with cached decorator usage and it's should pass build

**Comment by NullVoxPopuli** [comment]
:upside_down_face: 
image

**Comment by NullVoxPopuli** [comment]
Test success:
image

File: tests/scenarios/scenarios.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ async function lts_3_28(project: Project) {
   project.linkDevDependency('ember-data', { baseDir: __dirname, resolveName: 'ember-data' });
 }
 
-async function release(project: Project) {
+export async function release(project: Project) {
```
Comment (ef4) [inline tests/scenarios/scenarios.ts:42]: Instead of this, you can say `appScenarios.only('release')`

**Review by ef4 — APPROVED** [review]

File: tests/scenarios/scenarios.ts (hunk 1 above)
Comment (NullVoxPopuli) [reply tests/scenarios/scenarios.ts:42]: ooo that's handy, because I have other tests I'm working on that need just the release bit. thanks!

Comment (NullVoxPopuli) [reply tests/scenarios/scenarios.ts]: Updated!

**Pull request merged**